janusgraph-dist/docker/Dockerfile

```diff
@@ -73,7 +73,7 @@ RUN chmod 755 /usr/local/bin/docker-entrypoint.sh && \
     chmod 755 /usr/local/bin/load-initdb.sh && \
     chmod 755 /usr/bin/yq && \
     mkdir -p ${JANUS_INITDB_DIR} ${JANUS_CONFIG_DIR} ${JANUS_DATA_DIR} && \
-    chown -R janusgraph:janusgraph ${JANUS_HOME} ${JANUS_INITDB_DIR} ${JANUS_CONFIG_DIR} ${JANUS_DATA_DIR}
+    chown janusgraph:janusgraph ${JANUS_HOME} ${JANUS_INITDB_DIR} ${JANUS_CONFIG_DIR} ${JANUS_DATA_DIR}
```

Comment:
(nitpick) Not sure if this change is needed at all. Won't it be executed only once during image build process anyways? That said, I'm OK with removing `-R` flag here if that helps anyhow.

Reply:
It probably doesn't make any different, cuz in this case this will always be an empty folder, right?

Reply:
`${JANUS_HOME}` is not an empty folder on this step. It does contain some files when line 76 is executed

Reply:
Okay, thanks for pointing that out, I've re-added the `-R`.